CIP-0013/README.md

```diff
@@ -49,74 +57,79 @@ URIs for weighted stake pool lists provide alternatives to using a JSON file to
 
 The core implementation should follow the [BIP-21](https://github.com/bitcoin/bips/blob/master/bip-0021.mediawiki) standard (with `bitcoin:` replaced with `web+cardano:`)
 
-Rationale:
-- Use `cardano:` over `ada:` as other projects that implement this standard tend to take the project name over the currency name (this makes sense if we consider this protocol as a generic way for interacting with the blockchain through wallets - as opposed to a simple payment system)
-- Many wallets support multiple currencies. Following the same standard will ensure higher adoption of our protocol.
-
 Examples:
 ```
 <a href="web+cardano:Ae2tdPwUPEZ76BjmWDTS7poTekAvNqBjgfthF92pSLSDVpRVnLP7meaFhVd">Donate</a>
```

Comment:
> `<a href="web+cardano:Ae2tdPwUPEZ76BjmWDTS7poTekAvNqBjgfthF92pSLSDVpRVnLP7meaFhVd">Donate</a>`

Is there a reason why this URI schema has persisted to this point lacking the two forward slashes? (//)

Since we are looking to standardize this and prepare for future expansion, would it be safe to require that web+cardano:// always be present?

Further, should we consider a "pay to" _authority_ rather than defaulting to hoping that it is a valid address?

Reply:
> We are locked into supporting the payment links without an authority because for a long time that was the only implementation of these URIs that was getting professional consideration (someone suggested stake pool links in 2018 but nobody ever moved on it; timeline & links are in the PRs at the Discussion links).

> As far as I know Yoroi in particular has supported payment links with a bare path-query string that whole time: about 5 years now. If I had to tell them they needed to commit to honouring an authority prefix (let's call it //payment) I would therefore expect about 5 years to get a response.

Personally I think it's not too late to change the URI scheme to add an authority for payments, if that's what you think is best (I do). And while it is difficult to say whether there is a lot of people using payments URIs right now with Yoroi, I don't think it's a big deal to add this breaking change. But it'd be better of course if you try to get feedback from wallet providers first.

> how a wallet or perhaps a dApp would handle a URI with an invalid payment address would be up to the designers

Exactly